numpy/core/include/numpy/npy_common.h

```diff
@@ -1002,6 +1011,7 @@ typedef struct { npy_longdouble real, imag; } npy_clongdouble;
 #    define PyComplex256ArrType_Type PyCFloatArrType_Type
 #define NPY_FLOAT128_FMT NPY_FLOAT_FMT
 #define NPY_COMPLEX256_FMT NPY_CFLOAT_FMT
+#define NPY_FLOAT_BINFMT Intel_extended128
```

Comment:
These names are very confusing, because `NPY_FLOAT128_FMT` is a string for format strings, yet `NPY_FLOAT_BINFMT` is a preprocessor token that isn't valid unless concatenated

Reply:
Yeah, I could call it `NPY_FLOAT_BINFMT_NAME` or similar to lessen the confusion.

Reply:
Perhaps it should be `NPY_LONGDOUBLE_FMT_NAME`?